ReactiveObjCBridgeTests/ObjectiveCBridgingSpec.swift

```diff
@@ -300,5 +316,20 @@ class ObjectiveCBridgingSpec: QuickSpec {
 				}
 			}
 		}
+		
+		describe("RACSubscriber.sendNext") {
```

Comment:
Although this test is mainly validating the Swift compiler's behavior, it seems important to have this somewhere, and the `ReactiveObjC` repo seemed slightly awkward because it doesn't currently have any Swift code or tests.